pages/good-page.js

```diff
@@ -1,5 +1,6 @@
 import React, { useState } from "react";
```

Comment:
hey Magnus, what is the difference between your good page and bad page? This looks like you have written the same code twice. I would say try to combine into one page or at least one component which can address both cases.